libs/common-services/src/test/java/uk/gov/di/ipv/core/library/service/ConfigServiceTest.java

```diff
@@ -405,15 +370,29 @@ void shouldReturnListOfClientRedirectUrls(String testDataSet, String featureSet)
     }
 
     @ParameterizedTest
-    @CsvSource({"FEATURE_FLAGS,", "FEATURE_FLAGS,FS07"})
+    @CsvSource({"FEATURE_FLAGS,", "FEATURE_FLAGS,'FS07,DS01'", "FEATURE_FLAGS,'DS01,FS07'"})
     void shouldGetNamedFeatureFlag(String testDataSet, String featureSet) {
         environmentVariables.set("ENVIRONMENT", "test");
-        configService.setFeatureSet(featureSet);
+        List<String> fsList =
+                featureSet != null ? List.of(featureSet.split(",")) : Collections.emptyList();
```

Comment:
Maybe move this logic to `getFeatureSet` and use it?

...perhaps we need a helper somewhere that parses a featureSet string into a list (which you can use in the main code too).